Microsoft.HealthVault/Connection/HealthVaultConnectionBase.cs

```diff
@@ -178,6 +178,11 @@ protected HealthVaultConnectionBase(IServiceLocator serviceLocator)
             this.SessionCredential = await sessionCredentialClient.GetSessionCredentialAsync(token).ConfigureAwait(false);
         }
 
+        Task IConnectionInternal.RefreshSessionCredentialAsync(CancellationToken token)
+        {
+            return this.RefreshSessionCredentialAsync(token);
```

Comment:
Should we move the lock into the protected RefreshSessionCredentialAsync method?

---
In reply to: [115836683](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/27#discussion_r115836683) [](ancestors = 115836683,115805026,115799063)